packages/ui/src/components/Button/index.tsx

```diff
@@ -34,6 +34,7 @@ function Button(props: ButtonProps): JSX.Element {
     }
 
     return name;
+    // @ts-ignore
   }, [props.intent, props.icon]);
```

Comment:
Previous typescript error:
![image](https://user-images.githubusercontent.com/15877323/81805390-0e9c4780-951b-11ea-83a7-bf9e26faad03.png)

Reply:
@Sansossio I think I have to check this behavior, anyway lets leave the ignore comment so you are allowed to proceed!